src/panel/poi/blocks/Phone.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class PhoneBlock extends React.Component {
```

Comment:
Don't you want to use functional components for new stuff, especially when it's so elementary?

Reply:
Of course, it's done